collectors/bzimport/convertors.py

```diff
@@ -346,10 +346,20 @@ def save(self):
         # we don't query this flaw during the process
         with transaction.atomic():
             for part in self.all_parts:
+                kwargs = {}
+
+                # we want to store the original timestamps
+                # so we turn off assigning the automatic ones
                 if isinstance(part, TrackingMixin):
-                    part.save(auto_timestamps=False)
-                else:
-                    part.save()
+                    kwargs["auto_timestamps"] = False
+
+                # we want to store all the data fetched by the collector
+                # so we suppress the exception raising in favor of alerts
+                if isinstance(part, AlertMixin):
+                    kwargs["raise_validation_error"] = False
```

Comment:
I might be wrong but I think you don't need to be aware of the instance type, you can just always pass these as kwargs and they'll be ignored if it's not an `AlertMixin` / `TrackingMixin`

Reply:
If I leave it unconditionally (always) it fails at https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/mixins.py#L50 with `got an unexpected keyword argument` so I assume that `models.Model.save()` does not accept unexpected arguments.